spec/liquid_tags/unified_embed_spec.rb

```diff
@@ -68,17 +83,17 @@
         .to eq(WikipediaTag)
     end
 
-    valid_youtube_url_formats.each do |url|
-      it "returns YoutubeTag for a valid youtube url format" do
+    valid_vimeo_url_formats.each do |url|
```

Comment:
No changes were made to the Youtube and Vimeo specs, but I alphabetized them properly.